plugins/obs-ffmpeg/obs-ffmpeg-vaapi.c

```diff
@@ -543,7 +543,7 @@ static bool vaapi_encode(void *data, struct encoder_frame *frame,
 	}
 
 	ret = avcodec_send_frame(enc->context, hwframe);
-	if (ret == 0)
+	if (ret == 0 || AVERROR(EAGAIN))
```

Comment:
Shouldn't this be `ret == AVERROR(EAGAIN)`?

Reply:
That is what I thought I typed, fixed and retested. Thanks.